### PR TITLE
Poetry setup

### PR DESCRIPTION
**Pull request by GoberInfinity**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+[tool.poetry]
+name = "otter-welcome-buddy"
+version = "0.1.0"
+description = ""
+authors = ["goberinfinity <reyesfragosoroberto@gmail.com>"]
```
Comment (akotadi) [inline pyproject.toml]: Maybe with this one, we can put the nutria email in general.

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+[tool.poetry]
+name = "otter-welcome-buddy"
```
Comment (akotadi) [inline pyproject.toml:2]: I think *otter buddy* is better for being shorter, but if you prefer we can update that after deprecating the other repo.

**Review by akotadi — APPROVED** [review]
LGTM

File: pyproject.toml (hunk 2 above)
Comment (GoberInfinity) [reply pyproject.toml:2]: I will use this bot as a blueprint to make easier our deployment process, after that we can begin the migrations and proper modification to integrate our old bot.

**Pull request merged**